src/frontend/shortlist/src/components/layout/shortlists/ShortlistSchoolIcon.vue

```diff
@@ -10,10 +10,18 @@ export default {
   >
     <img class="school-icon-img" :src="schoolData.img" />
     <div class="school-icon-name shortlist-fnt-serif-bold">
+      <!-- Old school data format -->
+      <div v-if="schoolData.name">
       {{ schoolData.name }}
+      </div>
+      <!-- New school data format -->
+      <div v-else>
```

Comment:
I know more info will be added here (pulled from the api later), but this looks good for now.